Benchmarks/PrintingStyle.swift

```diff
@@ -0,0 +1,130 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+enum PrintingStyle {
+    case plainText
+    case json
+}
+
+extension BenchmarkResults {
+    func print(using style: PrintingStyle) {
+        switch style {
+        case .plainText:
+            printAsPlainText()
+        case .json:
+            printAsJSON()
+        }
+    }
+
+    private func printAsPlainText() {
+        let average = self.interpretedTimings.average
```

Comment:
(minor, feel free to address later)

Please drop explicit `self.` for consistency, and to reduce noise (e.g. in `Array.standardDeviation`):
```suggestion
        let average = interpretedTimings.average
```

Also other occurrences in `BenchmarkConfiguration.printAsPlainText`, `Array.average`, `Array.standardDeviation`.